crates/engine/src/resolver/graphql/request/execute.rs

```diff
@@ -136,7 +136,7 @@ pub(crate) async fn execute_subgraph_request<'ctx, R: Runtime>(
                         }
                     }
 
-                    if status.is_server_error() || status == http::StatusCode::TOO_MANY_REQUESTS {
```

Comment:
The problem is that this interfers with rate limit and retry logic, so it's that simple

Reply:
2nd try: https://github.com/grafbase/grafbase/pull/3584/commits/01b7fc28c99d8fba685e14fa06810490cb6bace1